Add a doc about Module Self Configurator feature (#1558)
PrestaShop/docs: modules/configure-with-cli.md

{{% notice info %}}
Please note that the third argument `<configfilepath>` is optional. If not set, PrestaShop will try to find a file named `self_config.yml` in the module's folder (and sub-folders).
{{% /notice %}}

## Apply configuration file on a specific shop or group

The Prestashop `prestashop:module` command allows you to restrict its actions on a specific shop (or group).

To do so, add an `--id_shop` or an `--id_shop_group` argument to the command:

```bash
php bin/console prestashop:module configure <modulename> <configfilepath> --id_shop=<id>
php bin/console prestashop:module configure <modulename> <configfilepath> --id_shop_group=<id>
```

## Configuration file reference

### Change values in the Configuration table

The Self Configurator feature allows editing values in the `Configuration` table from the configuration file. 

To do so, add a `configuration:` line, choose if you want to update or delete a configuration value, and add the configuration key and its value like in those examples:

```yaml
configuration:
    update:
        PAYPAL_SANDBOX: 1
```

```yaml
configuration:
    delete:
        - "PAYPAL_ONBOARDING"
```

{{% notice info %}}

> Please note that you can only `update` or `delete` values: `create` values are only done when installing the module.

Reviewer: You can add value. The update will create if needed.

Author: thanks, fixed 👍

> Please note that you can only `update` or `delete` values: `create` values are only done when installing the module.

Author: ```suggestion
Please note that you can `create` values by specifying them in the `update` section.
```
